src/SymbolProvider/include/SymbolProvider/ModulePathAndBuildId.h

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2023 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef SYMBOL_PROVIDER_MODULE_PATH_AND_BUILD_ID_H_
+#define SYMBOL_PROVIDER_MODULE_PATH_AND_BUILD_ID_H_
+
+#include <string>
+
+namespace orbit_symbol_provider {
+
+struct ModulePathAndBuildId {
+  std::string module_path;
+  std::string build_id;
+
+  [[nodiscard]] friend bool operator==(const ModulePathAndBuildId& lhs,
+                                       const ModulePathAndBuildId& rhs) {
+    return lhs.module_path == rhs.module_path && lhs.build_id == lhs.build_id;
+  }
+  [[nodiscard]] friend bool operator!=(const ModulePathAndBuildId& lhs,
+                                       const ModulePathAndBuildId& rhs) {
+    return !(lhs == rhs);
+  }
+
+  template <typename H>
```

Comment:
I guess this should have the `VerifyTypeImplementsAbslHashCorrectly` test.

Reply:
Done.

Reply:
@dpallotti That was actually a very good advice, as it catched a bug in the implementation of equals, that all of us missed.

Reply:
Haha yeah, that's hard to notice while reviewing. Good to see that that test is useful!